report-ng/app/src/main.ts

```diff
@@ -32,6 +32,7 @@ export function configure(aurelia: Aurelia) {
     aurelia.use
         .standardConfiguration()
         .plugin(PLATFORM.moduleName('@aurelia-mdc-web/base'))
+        .plugin(PLATFORM.moduleName('@aurelia-mdc-web/button'))
```

Comment:
Not used `mdc-button`

Reply:
`mdc-button` fragments have now been removed.